include/LCUI/LCUI.h

```diff
@@ -32,6 +32,7 @@
 #ifndef LCUI_H
 #define LCUI_H
 
+#include <LCUI_Build.h>

```

Comment:
> Add "#include <LCUI_Build.h>" to "include/LCUI.h" to avoid issues when include order is wrong

LCUI_Build.h should always be the first one to be included, I think LCUI.h does not need to be modified.

I have the following improvements, how do you feel?

files:

``` diff
+ include/LCUI.h
```

include/LCUI.h：

``` c
#include <LCUI_Build.h>
#include <LCUI/LCUI.h>
```
After this modification, we can include the header file in a new way:

``` diff
- #include <LCUI_Build.h>
- #include <LCUI/LCUI.h>
+#include <LCUI.h>
```

Reply:
That seems like a good idea. Similar to how windows handles all important includes with Windows.h

Reply:
@toluschr Ok, since you and I agree with these improvements, please replace your current changes with the above.